quickstarts/microsoft.maintenance/maintenance-configuration-assignments/metadata.json

```
@@ -0,0 +1,10 @@
+{
+    "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+    "type": "QuickStart",
+    "itemDisplayName": "Create maintenance configuration assignments",
```

We can just say _configuration assignments_ to avoid confusion b/w maintenance configurations and configuration assignments - similar in below lines
```suggestion
    "itemDisplayName": "Create configuration assignments",
```